src/ui/MobileControls.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+
+export function MobileControls() {
```

Comment:
Could the fill & stroke be changed to `white`? `currentColor` can be weird sometimes.

Reply:
Also the paths can use some minifcation, it's rounding into a lot of precision which is taking up valuable bytes and 9/10 not even needed/noticeable.